app/models/person.rb

```diff
@@ -94,23 +94,22 @@ def my_protocols(for_myself = true)
     prot_subs = protocol_subscriptions.active.joins(
       :protocol
     ).joins(
-      'FULL JOIN one_time_responses ON one_time_responses.protocol_id = protocols.id'
+      'LEFT JOIN one_time_responses ON one_time_responses.protocol_id = protocols.id'
```

Comment:
added a spec for this to prevent it in the future?

Reply:
Because of the where conditions that we put in later down the line, it returns the same exact result and functionally, these queries are identical. It's just that I don't want to have to look at this later and wonder why we use a FULL join instead of a LEFT one, when just using a LEFT one should have been enough.